content/client-lib-development-guide/features.textile

```diff
@@ -606,6 +612,13 @@ h4. ClientOptions
 *** @(TO3k3)@ @realtimeHost@ string - for development environments only; allows a non-default Ably Realtime host to be specified
 *** @(TO3k4)@ @port@ integer - for development environments only; allows a non-default Ably non-TLS port to be specified
 *** @(TO3k5)@ @tlsPort@ integer - for development environments only; allows a non-default Ably TLS port to be specified
+** @(TO3l)@ The follow attributes, if set, are used to change the default behaviour of the libary:
+*** @(TO3l1)@ @disconnectedRetryFrequency@ integer - default 15,000 (15s). When the connection is in the @DISCONNECTED@ state, this delay in milliseconds controls how frequently the client library attempts to reconnect automatically
+*** @(TO3l2)@ @suspendedRetryFrequency@ integer - default 30,000 (30s). When the connection is in the @SUSPENDED@ state, this delay in milliseconds controls how frequently the client library attempts to reconnect automatically
+*** @(TO3l3)@ @httpOpenTimeout@ integer - default 4,000 (4s). Timeout for opening the connection, available in the client library if supported by the transport
```

Comment:
I deliberated over this one, and decided to leave it in only because most HTTP client libraries that I took a quick look at supported separate timeout values for connect & response. The reason I think it's useful is that if an HTTP request is not opened within 4s, then it's quite possible there is a problem connecting as the HTTP server is probably not "alive" and using a fallback host is therefore sensible.  Having only one timeout means we'd need to configure the two values to be the same, so an unresponsive HTTP server could hold up a single REST HTTP request for 15s and therefore prevent any fallbacks being used at all because the overall `httpMaxRetryDuration` would have been exceeded.  Definitely one for discussion though.